includes/wc-admin-update-functions.php

```diff
@@ -61,3 +62,17 @@ function wc_admin_update_0230_rename_gross_total() {
 function wc_admin_update_0230_db_version() {
 	Installer::update_db_version( '0.23.0' );
 }
+
+/**
+ * Remove the note unsnoozing scheduled action.
+ */
+function wc_admin_update_0251_remove_unsnooze_action() {
+	wc_unschedule_action( WC_Admin_Notes::UNSNOOZE_HOOK, array(), WC_Admin_Notes::QUEUE_GROUP );
```

Comment:
I'm also getting a warning here:
```
PHP Notice:  wc_unschedule_action is <strong>deprecated</strong> since version 2.1.0! Use as_unschedule_action() instead.
```